docs/guide/quickstart.rst

```diff
@@ -0,0 +1,295 @@
+:orphan:
+
+.. currentmodule:: discord
+
+.. _guide_quickstart:
+
+Quickstart
+===========
+
+This page will contain a simple explanation of how to make a basic bot using the commands extension "ext.commands".
+Before you begin, make sure to install discord.py - see :ref:`guide_intro_installation` for more information.
+
+Obtaining a Bot Token
+----------------------
+
+You will need a bot application before being able to run a bot. A step-by-step tutorial with screenshots follows:
+
+1. Go to the `Discord Developer Portal <https://discord.com/developers/applications/>`_.
+
+2. Create a new application if you have not made one already:
+    .. image:: /images/discord_create_app_button.png
+        :scale: 90 %
+
+3. Navigate over to the ``bot`` tab:
+    .. image:: /images/discord_bot_tab.png
+        :scale: 80%
+
+4. Create a new Bot user if you have not done so already:
+    .. image:: /images/discord_create_bot_user.png
+
+5. If you need these intents, scroll down and click on these check boxes:
+    .. image:: /images/discord_privileged_intents.png
+
+6. Your bot token will only be retrievable upon creation and looks like this:
+    .. image:: /images/discord_bot_token.png
+
+6a. Otherwise it will look like this:
+    .. image:: /images/discord_bot_token_no_copy.png
+
+.. danger::
+
+    This token is **VERY IMPORTANT** and you should treat it like the password to your bot. Keep it as secure as possible and never give this to anyone.
+    If you ever leak it, you can click the ``Regenerate`` button to forcibly invalidate all copies of the token. This will also terminate all running instances of this bot.
+
+.. _guide_quickstart_client_secret:
+
+.. warning::
+
+    You may be thinking to use the ``Client Secret`` for your bot token. This is **NOT** what you are looking for, you need the bot **Token**, which is a completely different format.
+
+    .. image:: /images/discord_client_secret_big_nono.png
+        :scale: 90 %
+
+A Simple Bot
+-------------
+
+This step-by-step walk-through will show you how to make a bot using the commands framework.
+
+.. note::
+
+    This walk-through does not cover application commands (slash commands). For a detailed walk-through of application commands, see _guide_app_commands.
+
+
+1. Create a new Python file in the folder you want to work in.
+
+    .. warning::
+
+        Do not name the file ``discord.py``, this will cause conflicts with the discord.py library. Also, do not create a sub-folder in your project named ``discord``, as that too will cause conflicts.
```

Comment:
```suggestion
        Do not name the file ``discord.py``, as this will cause conflicts with the discord.py library. Also, do not create a sub-folder in your project named ``discord``, as that too will cause conflicts.
```